tests/product/constants.py

```diff
@@ -16,16 +16,21 @@
 Module defining constants global to the product tests
 """
 
+import json
 import os
 import sys
 
 import prestoadmin
 from prestoadmin import main_dir
 
+BASE_IMAGES_TAG_CONFIG = 'base-images-tag.json'
+
 try:
-    BASE_IMAGES_TAG = os.environ['BASE_IMAGES_TAG']
+    with open(os.path.join(main_dir, BASE_IMAGES_TAG_CONFIG)) as bit_conf:
+        bit_json = json.load(bit_conf)
```

Comment:
don't abbreviate as 'bit'-- it's confusing because that's already a word.  You could call this tag_conf and tag_json.